src/ui_widgets/dropdown.gd

```diff
@@ -5,6 +5,8 @@ extends HBoxContainer
 
 @export var values: PackedStringArray
 @export var restricted := true
+## If set to a value greater than 0, the popup will have a fixed height, and scrolling will activate if the content exceeds it.
+@export var popup_panel_height := 0
```

Comment:
Have it at -1.0 like the default value of the argument.

Also, is there actually any reason to have width/height be floats?

Reply:
> Also, is there actually any reason to have width/height be floats?

For the `min_size`, we're directly using a value like `size.x`, which is a float. When converted to an integer, this causes a loss of precision. However, the impact on our use case will be negligible, and it will only result in warnings in the editor `Narrowing conversion: float is converted to int and loses precision` if we don't manually handle the conversion.

I would suggest keeping it as a float.